spec/features/system_admin/uploads/delete_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Delete a file" do
+  let!(:user) { create(:user, :system_admin) }
```

Comment:
We try to wrap this kind of stuff in a descriptive method and call it in the `background` step. We ecnourage devs to avoid `let` in feature files, but it's up to the developer.